ui/imports/shared/views/chat/ChannelIdentifierView.qml

```diff
@@ -32,7 +32,7 @@ Column {
         icon {
             width: 120
             height: 120
-            color: root.chatColor
+            color: root.chatType === Constants.chatType.oneToOne ? Utils.colorForPubkey(root.chatId) : root.chatColor
```

Comment:
This one you maybe do that condition in Nim instead of in QML. You can populate `chatColor` with the right color if it's a 1-1 chat. I think you can get that value from the Contact?

Reply:
Unfortunately not, I wanted to do it that way initially, but the color palette we use for mapping lies in StatusQ.
I think that's good, because different frontends/themes/styles may interpret colorId differently.